packaging/java/pom.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.github.paulfitz</groupId>
+    <artifactId>daff</artifactId>
+    <version>1.3.48</version>

```

Comment:
Will this need updating each time the version changes?

Reply:
You are right. Honestly I wasn't comfortable using a static version. Now the version is read from package.json and it is then passed to mvn as parameter.

Reply:
I saw you made updates, thanks. I tried running `mvn jreleaser:config` and got `org.jreleaser.model.JReleaserException: Version ${parameter.version} does not follow the semver spec`. Is something extra needed to pass in a parameter?

Reply:
Oops! I forgot to update the mvn command in RELEASE.md. I just pushed a commit that fixes this. The property is now called revision. I changed it because I was getting a warning and Google led me to this snippet from the [Maven documentation on CI-friendly versions](https://maven.apache.org/maven-ci-friendly.html):

> Starting with Maven 3.5.0-beta-1 you can use the ${revision}, ${sha1} and/or ${changelist} as placeholders for the version in your pom file.